### ISSUE TITLE
real-test failures on Power9

### ISSUE THREAD

**Issue by pjh40** [flang-compiler/f18#211]
I suspect that this is similar to the problems reported in issue #148, but, for completeness, real-test has 480 failures on an IBM Power9 Linux system:
```
uname: Linux cn2024 4.14.0-49.10.1.el7a.ppc64le #1 SMP Thu Jul 5 17:54:38 UTC 2018 ppc64le ppc64le ppc64le GNU/Linux
GCC 8.2.0: Target: powerpc64le-unknown-linux-gnu
```

[real-fail.txt](https://github.com/flang-compiler/f18/files/2469157/real-fail.txt)

**Comment by klausler** [comment]
Fixed by PR#266

**Comment by pjh40** [comment]
All tests pass on Power9 release build, thanks!
